express/blocks/template-list/template-list.css

```
@@ -356,6 +356,17 @@ main .template-list.sixcols .template.placeholder {
   margin: 15px auto;
 }
 
+/* template-list-scrollbutton */
+main .template-list.sixcols .template-list-scrollbutton {
+  padding: 7px 8px;
+  position: fixed;
+  bottom: 0;
+  right: 0;
+  z-index: 1;
```

```suggestion
  z-index: 3;
```
Increase z-index to make sure scroll button stays above floating CTA button.